### PR TITLE
Initialize Android Campaign Classic 2.0 project

### PR DESCRIPTION
**Pull request by spoorthipujariadobe**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: .github/workflows/maven-release.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle
+      - name: Import GPG key
+        env:
+          GPG_SECRET_KEYS: ${{ secrets.GPG_SECRET_KEYS }}
+          GPG_OWNERTRUST: ${{ secrets.GPG_OWNERTRUST }}
+        run: |
+          echo $GPG_SECRET_KEYS | base64 --decode | gpg --import --no-tty --batch --yes
+          echo $GPG_OWNERTRUST | base64 --decode | gpg --import-ownertrust --no-tty --batch --yes
+      - name: Publish to maven central staging repository
```
Comment (sbenedicadb) [inline .github/workflows/maven-release.yml:29]: name should not indicate we are publishing to staging

**Review by sbenedicadb — CHANGES_REQUESTED** [review]
small typo

**Review by PravinPK — APPROVED** [review]
Lets go!!

**Review by sbenedicadb — APPROVED** [review]
looks good 👍

**Pull request merged**